app/templates/components/repo-show-tabs.hbs

```diff
@@ -27,6 +27,15 @@
       {{/link-to}}
     {{/if}}
   </li>
+  {{#if repo.slug}}
+    {{#if repo.cronJobs.length}}
```

Comment:
Did you find that the `.length` was necessary? I believe that an empty array is “falsy” in Handlebars.